src/libostree/ostree-repo-static-delta-compilation.c

```diff
@@ -997,6 +1015,18 @@ generate_delta_lowlatency (OstreeRepo                       *repo,
       const char *from_checksum = value;
       ContentRollsum *rollsum;
       ContentBsdiff *bsdiff;
+      gboolean from_world_readable = FALSE;
+      gboolean to_world_readable = FALSE;
+
+      if (!check_object_world_readable (repo, from_checksum, &from_world_readable, cancellable, error))
```

Comment:
I think a comment here explaining why non-world-readable files are being skipped would be good.